templates/flavors/etcd-disk/kustomization.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../default
+
+patches:
+  - target:
+      group: infrastructure.cluster.x-k8s.io
+      version: v1alpha1
+      kind: LinodeMachineTemplate
+      name: .*-control-plane
+    patch: |-
+      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+      kind: LinodeMachineTemplate
+      metadata:
+        name: ${CLUSTER_NAME}-control-plane
+      spec:
+        template:
+          spec:
+            dataDisks:
+              sdc:
+                label: etcd_disk
+                size: 10Gi
+  - target:
+      group: controlplane.cluster.x-k8s.io
+      version: v1beta1
+      kind: KubeadmControlPlane
+    patch: |-
+      apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+      kind: KubeadmControlPlane
+      metadata:
+        name: ${CLUSTER_NAME}-control-plane
+      spec:
+        kubeadmConfigSpec:
+          clusterConfiguration:
+            etcd:
+              local:
+                dataDir: /var/lib/etcd_data/etcd
+                extraArgs:
+                  quota-backend-bytes: "8589934592"
```

Comment:
do we still need this section if it is in the default flavor?

Reply:
We technically don't but I wanted to leave it in for clarity. I'm fine with removing it